newhelm/config.py

```diff
@@ -25,13 +25,14 @@ def write_default_config(dir: str = DEFAULT_CONFIG_DIR):
         shutil.copyfile(source_file, output_file)
 
 
-def load_secrets_from_config(
-    path: str = SECRETS_PATH, registry: SecretsRegistry = SECRETS
-) -> None:
-    """If the secrets file exists, use it to call SECRETS.set_values."""
-    if not os.path.exists(path):
-        # Nothing to load
-        return
+def load_secrets_from_config(path: str = SECRETS_PATH) -> RawSecrets:
+    """Load the toml file and verify it is shaped as expected."""
     with open(path, "rb") as f:
-        values = tomli.load(f)
-    registry.set_values(values)
+        data = tomli.load(f)
+    for values in data.values():
+        # Verify the config is shaped as expected.
+        assert isinstance(values, Mapping), "All keys should be in a [scope]."
+        for key, value in values.items():
+            assert isinstance(key, str)
+            assert isinstance(value, str)
```

Comment:
Very minor but wanted to make sure there weren't any situations where someone would want to do

```toml
mysecret = 123
```

Or more edge-case-like would be an API key or other secret that happens to not have any letters in it

Reply:
The downstream tooling in all 3 versions of secrets assumes `get_required` is returning a string, so this assert is ensuring that will work as expected. If you have a value that needs to be some other type, you can always convert the string.

> Or more edge-case-like would be an API key or other secret that happens to not have any letters in it

In toml you can (must?) surround string values with quotes. So even if your API key is `123` you can ensure it is a string with `"123"`.

Reply:
For sure. Only making sure it was intentional. Thanks.